core/src/main/scala/com/microsoft/azure/synapse/ml/logging/common/SparkHadoopUtils.scala

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.azure.synapse.ml.logging.common
+
+import org.apache.spark.SparkContext
+
+import com.microsoft.azure.synapse.ml.logging.SynapseMLLogging
+
+object SparkHadoopUtils {
+
+  def getHadoopConfig(key: String, SC: SparkContext): String = {
+    if (SC == null) {
+      ""
```

Comment:
retuning empty strings instead of throwing errors is confusing, please dont use special return values here or elsewhere, just throw an error

Reply:
ok